server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -718,9 +718,9 @@ In order to do that: `default=true`
 
 The web-admin server will return `404` response code when the default identity can not be found.
 
-=== Creating a user identity
+=== Creating a JMAP user-set identity
```

Comment:
Take out the user-set in the doc, just keep user (so other lines as well). It makes no sense in the doc